src/aliceVision/sfm/pipeline/ReconstructionEngine.hpp

```diff
@@ -81,6 +82,8 @@ class ReconstructionEngine
   std::string _outputFolder;
   /// Internal SfMData
   sfmData::SfMData _sfmData;
+  //Random engine
+  std::mt19937 _randomNumberGenerator;
```

Comment:
Shouldn't we initialize it with `std::random_device` and expose a method to set a custom seed?